pkg/util/util.go

```diff
@@ -77,7 +77,7 @@ func GetPollerStatuses() ([]PollerStatus, error) {
 	for _, p := range processes {
 		line, err := p.Cmdline()
 		if err != nil {
-			if !errors.Is(err, unix.EINVAL) {
+			if !errors.Is(err, unix.EINVAL) && !errors.Is(err, unix.ENOENT) {
```

Comment:
The important part is the `continue` on error. Logging the exact error is less important. I'm fine with adding another clause, but if logging the error continues to be problematic or confusing, we should change the logging to debug since it's mostly for debugging.